src/Exception/InvalidArgumentException.php

```diff
@@ -2,6 +2,7 @@
 
 namespace SimpleSAML\XMLSecurity\Exception;
 
+use InvalidArgumentException as BUILTIN_InvalidArgumentException;
```

Comment:
I have to admit I'm not a fan of the name here... It's just a nitpick, but maybe `BuiltInInvalidArgumentException`?

Reply:
Sure, I'll change it throughout